src/Repository/ArticleToCategory/Postgresql/PostgresqlCollection.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+namespace Repository\ArticleToCategory\Postgresql;
+
+class PostgresqlCollection extends \Maleficarum\Storage\Repository\Postgresql\Collection {
+    /**
+     * Return shard
+     *
+     * @param $collection
+     *
+     * @return \Maleficarum\Storage\Shard\ShardInterface
+     */
+    public function getShard($collection) {
+        return $this->getStorage()->fetchShard('Postgresql', ($this->shardSelector)($collection));
+    }
+
+    /**
+     * @see \Maleficarum\Storage\Repository\CollectionInterface.deleteAll()
+     */
+    public function deleteAll(\Maleficarum\Data\Collection\Persistable\AbstractCollection $collection): \Maleficarum\Storage\Repository\CollectionInterface {
+        $shard = $this->getShard($collection);
+
+        // Extract entity ids
+        $articleIds = [];
+        $categoryIds = [];
+        foreach ($collection->toArray() as $key => $col_element) {
```

Comment:
`$col_element`? co to za cudo? nie dość że brakuje snakeCase to jeszcze jakieś skracanie zmiennych przez co wychodzą koszmarki.